.npmrc

```diff
@@ -0,0 +1 @@
+force=true
```

Comment:
Don't want to leave this in long term, but makes it easier to deal with experimental releases here